From d12bf13660317a5e43f9fb700a11a34fe14bd977 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 27 Dec 2016 09:30:35 +0800
Subject: [PATCH 1446/5242] MLK-13684-2 video: mxsfb: add 'FB_MXC_OVERLAY'
 config

commit  3dcb5f533024fb7d7f205ca72fbaf9e37c3ce726 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add 'FB_MXC_OVERLAY' config to control overlay framebuffer
feature. And also add this config to defconfig.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/Kconfig |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/video/fbdev/mxc/Kconfig b/drivers/video/fbdev/mxc/Kconfig
index 0e3cfeb..e69e4e0 100644
--- a/drivers/video/fbdev/mxc/Kconfig
+++ b/drivers/video/fbdev/mxc/Kconfig
@@ -19,6 +19,13 @@ config FB_MXC_SYNC_PANEL
 	depends on MXC_IPU_V3
 	tristate "Synchronous Panel Framebuffer"
 
+config FB_MXC_OVERLAY
+	depends on FB_MXC
+	tristate "Overlay Framebuffer"
+	default n
+	help
+	  Enhanced LCD controller of MXC has overlay function.
+
 config FB_MXC_MIPI_DSI
 	tristate "MXC MIPI_DSI"
 	depends on FB_MXC_SYNC_PANEL
-- 
1.7.9.5

